solarwinds_apm/sampler.py

```diff
@@ -143,43 +140,8 @@ def calculate_liboboe_decision(
         xtraceoptions: Optional[XTraceOptions] = None,
     ) -> dict:
         """Calculates oboe trace decision based on parent span context and APM config."""
-        tracestring = None
```

Comment:
The main changes of this function are (1) moving the `if` of the `if-else` for lambda to the top and (2) actually using OboeAPI now.